Unable to install in grunt 0.4.1 via any methods

unable to install in grunt 0.4.1 via any methods:

```
➜  lrvick git:(develop) ✗ grunt --version
grunt-cli v0.1.7
grunt v0.4.1

➜  lrvick git:(develop) ✗ npm install grunt-imagemagick
npm http GET https://registry.npmjs.org/grunt-imagemagick
npm http 304 https://registry.npmjs.org/grunt-imagemagick
npm http GET https://registry.npmjs.org/fs
npm http GET https://registry.npmjs.org/path
npm http 304 https://registry.npmjs.org/path
npm WARN package.json path@0.4.9 No README.md file found!
npm http 404 https://registry.npmjs.org/fs
npm ERR! 404 'fs' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, or http url, or git url.

npm ERR! System Linux 3.6.11-gentoo
npm ERR! command "node" "/usr/bin/npm" "install" "grunt-imagemagick"
npm ERR! cwd /home/lrvick/Sources/example
npm ERR! node -v v0.8.21
npm ERR! npm -v 1.2.11
npm ERR! code E404
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /home/lrvick/Sources/example/npm-debug.log
npm ERR! not ok code 0

➜  lrvick git:(develop) ✗ npm install https://github.com/icagstrout/grunt-imagemagick.git
npm http GET https://github.com/icagstrout/grunt-imagemagick.git
npm http 200 https://github.com/icagstrout/grunt-imagemagick.git
npm ERR! not a package /home/lrvick/tmp/npm-9415/1372345185380-0.9913596250116825/tmp.tgz
npm ERR! Error: ENOENT, open '/home/lrvick/tmp/npm-9415/1372345185380-0.9913596250116825/package/package.json'
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Linux 3.6.11-gentoo
npm ERR! command "node" "/usr/bin/npm" "install" "https://github.com/icagstrout/grunt-imagemagick.git"
npm ERR! cwd /home/lrvick/Sources/example
npm ERR! node -v v0.8.21
npm ERR! npm -v 1.2.11
npm ERR! path /home/lrvick/tmp/npm-9415/1372345185380-0.9913596250116825/package/package.json
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /home/lrvick/Sources/example/npm-debug.log
npm ERR! not ok code 0

➜  lrvick git:(develop) ✗ npm install https://github.com/icagstrout/grunt-imagemagick/archive/master.zip
npm http GET https://github.com/icagstrout/grunt-imagemagick/archive/master.zip
npm http 200 https://github.com/icagstrout/grunt-imagemagick/archive/master.zip
npm ERR! not a package /home/lrvick/tmp/npm-9511/1372345206592-0.12830045307055116/tmp.tgz
npm ERR! Error: ENOENT, open '/home/lrvick/tmp/npm-9511/1372345206592-0.12830045307055116/package/package.json'
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Linux 3.6.11-gentoo
npm ERR! command "node" "/usr/bin/npm" "install" "https://github.com/icagstrout/grunt-imagemagick/archive/master.zip"
npm ERR! cwd /home/lrvick/Sources/example
npm ERR! node -v v0.8.21
npm ERR! npm -v 1.2.11
npm ERR! path /home/lrvick/tmp/npm-9511/1372345206592-0.12830045307055116/package/package.json
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /home/lrvick/Sources/example/npm-debug.log
npm ERR! not ok code 0
```

I've been having the same issue trying to get this installed.

``` shell
|ruby-1.9.3-p392| larryx in ~/Documents/test
○ → node --version
v0.8.23

|ruby-1.9.3-p392| larryx in ~/Documents/test
○ → npm --version
1.3.1

|ruby-1.9.3-p392| larryx in ~/Documents/test
○ → grunt --version
grunt-cli v0.1.9
grunt v0.4.1

|ruby-1.9.3-p392| larryx in ~/Documents/test
○ → npm install --save-dev grunt-imagemagick
npm http GET http://registry.npmjs.org/grunt-imagemagick
npm http 304 http://registry.npmjs.org/grunt-imagemagick
npm http GET http://registry.npmjs.org/fs
npm http GET http://registry.npmjs.org/path
npm http GET http://registry.npmjs.org/node-imagemagick
npm http 304 http://registry.npmjs.org/path
npm http 304 http://registry.npmjs.org/node-imagemagick
npm http 404 http://registry.npmjs.org/fs
npm ERR! 404 'fs' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, or http url, or git url.

npm ERR! System Darwin 12.4.0
npm ERR! command "node" "/Users/larrygordon/.nvm/v0.8.23/bin/npm" "install" "--save-dev" "grunt-imagemagick"
npm ERR! cwd /Users/larrygordon/Documents/test
npm ERR! node -v v0.8.23
npm ERR! npm -v 1.3.1
npm ERR! code E404
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/larrygordon/Documents/test/npm-debug.log
npm ERR! not ok code 0
```

Ditto. It appears that the [fs](https://npmjs.org/package/fs) is missing. A bug report has been submitted. https://github.com/isaacs/npm-www/issues/347

```
npm WARN package.json MyUVO@0.1.0 No README.md file found!
npm http GET https://registry.npmjs.org/grunt-imagemagick
npm http 304 https://registry.npmjs.org/grunt-imagemagick
npm http GET https://registry.npmjs.org/fs
npm http GET https://registry.npmjs.org/path
npm http GET https://registry.npmjs.org/node-imagemagick
npm http 404 https://registry.npmjs.org/fs
npm ERR! 404 'fs' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, or http url, or git url.

npm ERR! System Darwin 12.4.0
npm ERR! command "node" "/usr/local/bin/npm" "install" "grunt-imagemagick" "--save"
npm ERR! cwd ../..
npm ERR! node -v v0.10.4
npm ERR! npm -v 1.2.18
npm ERR! code E404
npm http 304 https://registry.npmjs.org/path
npm http 304 https://registry.npmjs.org/node-imagemagick
npm WARN package.json path@0.4.9 No README.md file found!
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     ../.../npm-debug.log
npm ERR! not ok code 0
```

--- a/package.json
+++ b/package.json
@@ -32,7 +32,6 @@
     "grunt": "~0.3.17"
   },
   "dependencies":{
-    "path":">= 0.0.1",
     "node-imagemagick":""
   },
   "keywords": [

